### DISCUSSION TITLE
How can I judge the current ENV at the same time of vue-cli or vite？

### DISCUSSION THREAD

**Discussion by UzkiS** [vuejs/vue-cli#6573] [Q&A]
How can I judge the current ENV at the same time of vue-cli or vite？
```
const isDevelopment =
  import.meta.env.MODE === "development" ||
  process.env.NODE_ENV === "development";
```
When I use vue-cli, the program prompts me that the import loader is missing，I try to use webpack import loader but it doesn't work

**Comment by Viggo95** [comment]
Maybe you can try using this👇
https://github.com/IndexXuan/vue-cli-plugin-vite
